src/Features/CSharp/Portable/Formatting/CSharpFormattingInteractionService.cs

```diff
@@ -108,15 +103,18 @@ public bool SupportsFormattingOnTypedCharacter(Document document, char ch)
             var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var span = textSpan ?? new TextSpan(0, root.FullSpan.Length);
             var formattingSpan = CommonFormattingHelpers.GetFormattingSpan(root, span);
-            var options = documentOptions ?? await document.GetDocumentOptionsWithInferredIndentationAsync(
-                explicitFormat: true, _indentationManagerService, cancellationToken).ConfigureAwait(false);
-
+            var options = documentOptions;
```

Comment:
Why the local versus just reassigning to the parameter?

Reply:
(ditto other points in the file)

Reply:
I was avoiding unnecessary changes. The local already existed, so see no benefit to changing as part of this PR.